kubectl-fdb/cmd/cordon.go

```diff
@@ -116,18 +139,15 @@ kubectl fdb cordon -c cluster --node-selector machine=a,disk=fast
 	cmd.Flags().StringP("fdb-cluster", "c", "", "evacuate process group(s) from the provided cluster.")
 	cmd.Flags().StringToStringVarP(&nodeSelectors, "node-selector", "", nil, "node-selector to select all nodes that should be cordoned. Can't be used with specific nodes.")
 	cmd.Flags().BoolP("exclusion", "e", true, "define if the process groups should be removed with exclusion.")
-	err := cmd.MarkFlagRequired("fdb-cluster")
-	if err != nil {
-		log.Fatal(err)
-	}
-
+	cmd.Flags().Bool("use-custom-label", false, "define if the process groups should be removed using label instead of cluster name.")
+	cmd.Flags().StringP("custom-label", "l", "fdb-cluster-name", "evacuate process group(s) from the provided cluster.")
```

Comment:
```suggestion
	cmd.Flags().StringP("custom-label", "l", fdbv1beta2. FDBClusterLabel, "evacuate process group(s) from the provided cluster.")
```

We should use the default label here if we define a default value for the flag. If we want to allow multiple custom labels we should make use of `StringArray` instead of a String and doing some custom parsing.